client/components/marketing-survey/cancel-purchase-form/step-components/educational-content-step.tsx

```diff
@@ -0,0 +1,330 @@
+						) }
+					</p>
+					<ul className="cancel-purchase-form__edu-icon-list">
+						<li>
+							<div className="cancel-purchase-form__edu-icon">
+								<MaterialIcon icon="power" size={ 24 } />
+							</div>
+							<div>
+								{ translate(
+									'Uninstall any unnecessary plugins. Sites with too many third-party plugins installed can take longer to load.'
+								) }
+							</div>
+						</li>
+						<li>
+							<div className="cancel-purchase-form__edu-icon">
+								<MaterialIcon icon="palette" size={ 24 } />
+							</div>
+							<div>
+								{ translate(
+									'Make sure you’re using a fast theme. Some themes come packed with extra features that can slow down your site.'
+								) }
+							</div>
+						</li>
+						<li>
+							<div className="cancel-purchase-form__edu-icon">
+								<MaterialIcon icon="photo_library" size={ 24 } />
+							</div>
+							<div>
+								{ translate(
+									'Optimize your images before uploading them. Unnecessarily large image files take longer to load.'
+								) }
+							</div>
+						</li>
+						<li>
+							<div className="cancel-purchase-form__edu-icon">
+								<MaterialIcon icon="library_books" size={ 24 } />
+							</div>
+							<div>
+								{ translate(
+									'Read more about how to improve your site’s speed and performance {{link}}here{{/link}}.',
+									{
+										components: {
+											link: <Button href="https://wordpress.com/support/site-speed/" />,
+										},
+									}
+								) }
+							</div>
+						</li>
+					</ul>
+				</Content>
+			);
+		case 'education:free-domain':
+			return (
+				<Content
+					image={ imgFreeDomain }
+					title={ translate( 'Step-by-step guide to get a free domain' ) }
+					onDecline={ props.onDecline }
+				>
+					<p>
+						{ translate(
+							'You can register a free domain name for one year with the purchase of any WordPress.com annual or two-year plan!'
+						) }
+					</p>
+					<ul>
+						<li>
+							{ translate(
+								'Go to Upgrades → Domains and click {{link}}Add a Domain{{/link}} to register your plans free domain',
+								{
+									components: {
+										link: <Button href={ `/domains/add/${ site.slug }` } isLink />,
+									},
+								}
+							) }
+						</li>
+						<li>
+							{ translate(
+								'Choose from all popular extensions including .com, .org, .net, .shop, and .blog.'
+							) }
+						</li>
+						<li>
+							{ translate(
+								'Make it your site’s primary domain (no “WordPress.com” in the address!)'
+							) }
+						</li>
+						<li>
+							{ translate( 'After the first year, your domain will renew at the regular price.' ) }
+						</li>
+						<li>
+							{ translate(
+								'If you have trouble claiming your free domain, contact us and we’ll assist you.'
+							) }
+						</li>
+						<li>
+							{ translate( 'Read more about domains {{link}}here{{/link}}.', {
```

Comment:
This button has unexpected styling
<img width="631" alt="Screenshot 2022-11-14 at 14 31 24" src="https://user-images.githubusercontent.com/82778/201660904-1d6b5a30-bcde-4be3-aa9f-1d46ebf9d3c2.png">

Reply:
Fixed!
<img width="581" alt="Cancel_Purchase_‹_Taggon_on_wordpress_—_WordPress_com" src="https://user-images.githubusercontent.com/212034/201855941-eebd4d25-4747-43db-91c2-20b6d9dab6f9.png">